### PR TITLE
Fix new line problem on database_host value

### PR DESCRIPTION
**Pull request by winggundamth**
## Description of the change

Add a dash to make it not have a new line.

## Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).

### CODE REVIEW

File: charts/backstage/templates/app-config.yaml [hunk 1]
```diff
@@ -76,7 +76,9 @@ stringData:
 
     auth:
       # see https://backstage.io/docs/auth/ to learn about auth providers
-      providers: {}
+      {{- with .Values.backstage.auth }}
+        {{- toYaml . | nindent 6 }}
+      {{- end }}
```
Comment (tumido) [inline charts/backstage/templates/app-config.yaml:81]: This was probably not meant to be included in this PR.

Comment (winggundamth) [reply charts/backstage/templates/app-config.yaml:81]: Arrr I forgot that I pushed all the stuff in the main branch. I will close this PR and open a new PR

Comment (winggundamth) [reply charts/backstage/templates/app-config.yaml]: I reverted back

**Review by tumido — APPROVED** [review]
/lgtm
/approve

:+1:

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**